app/services/brewerydb_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'httparty'
+require 'httplog'
+class BrewerydbService
+  include HTTParty
+  base_uri 'api.brewerydb.com'
```

Comment:
@edman193 
What do you think if we add a environment variable for the base_uri of the brewery api?